src/charm.py

```diff
@@ -11,11 +11,12 @@
 from charms.traefik_k8s.v1.ingress import IngressPerAppRequirer
 from ops.charm import CharmBase, ConfigChangedEvent
 from ops.main import main
-from ops.model import ActiveStatus, BlockedStatus, Container
+from ops.model import ActiveStatus, BlockedStatus, Container, StatusBase
 
 from charm_state import CharmState
-from consts import FLASK_APP_PORT, FLASK_CONTAINER_NAME, FLASK_SERVICE_NAME
-from exceptions import WebserverConfigInvalidError
+from consts import FLASK_CONTAINER_NAME, FLASK_SERVICE_NAME
```

Comment:
Very nitpicky, though I should ask - constants is being abbreviated to consts, whereas exceptions remains (and contants is 9 chars, exceptions 10) - can the naming be consistent like either abbreviate all that can be abbreviated or abbreviate none of them?

Reply:
Gotcha. I will update the module name `consts` to `constants` in the next PR https://github.com/canonical/flask-k8s-operator/pulls since this PR is merging.